ci/test.lua

```diff
@@ -189,39 +189,67 @@ local function ensure_title_screen()
                          dfhack.gui.getCurFocus(true)[1]))
 end
 
-local function is_fortress(focus_string)
-    focus_string = focus_string or dfhack.gui.getCurFocus(true)
-    return focus_string == 'dwarfmode/Default'
+local function is_fortress()
+    return dfhack.gui.matchFocusString('dwarfmode/Default')
```

Comment:
I think a strict match is better than a prefix match here. We don't want to be stuck in a submenu unexpectedly.

Reply:
`dwarfmode/Default` *is* a strict match here. the "`/Default`" part indicates that no other panel is covering the map

Reply:
Right. Good point.